File: 프로그래머스/0/120829. 각도기/각도기.java
class Solution {
    public int solution(int angle) {
        /*
        1 == 0 < 90 예각
        2 == 90 직각
        3 == < 180 둔각
        4 == 180 평각
        */
        int answer = 0;

        if (angle < 90) {
            answer = 1;
        } else if (angle == 90) {
            answer = 2;
        } else if (angle < 180) {
            answer = 3;
        } else {
            answer = 4;
        }
        return answer;
    }
}

File: 프로그래머스/0/120829. 각도기/각도기.py
def solution(angle):
    # 1 0도 초과 90도 미만은 예각, 
    # 2 90도는 직각, 
    # 3 90도 초과 180도 미만은 둔각 
    # 4 180도는 평각
    if angle > 0 and angle < 90:
        return 1
    elif angle == 90:
        return 2
    elif angle > 90 and angle < 180:
        return 3
    elif angle == 180:
        return 4